packages/roosterjs-editor-api/lib/format/rotateImage.ts

```diff
@@ -0,0 +1,16 @@
+import { ChangeSource } from 'roosterjs-editor-types';
+import { Editor } from 'roosterjs-editor-core';
+
+/**
+ * Rotate an image
+ * @param editor The editor instance
+ * @param image The selected image file
+ * @param angle The degree at which to rotate the image from it's center
+ */
+export default function rotateImage(editor: Editor, image: HTMLImageElement, angle: number): void {
```

Comment:
question: for an image already rotated, when we rotate again, should we get a combined a result or a overwritten one?

Reply:
I previously had it rotating based on the combined result, but I realize that would be very restricting depending on how our consumers want to rotate the image. 

For example, in OWA, we rotate dynamically based on where the mouse is. In this case we would want to update the rotation on the fly at whatever point the mouse is at. Other consumers may have different scenarios for how they want to handle this and doing it this way would allow them the flexibility to do that

Reply:
In OWA, if we do rotation on the fly, does it mean the function addUndoSnapshot() will be called multiple times when rotate? Does it mean user need to Ctrl+Z multiple times when they want to undo the rotation?